src/hardware/motor.cpp

```diff
@@ -31,7 +31,9 @@ StepperMotor::StepperMotor() {
 
 // Returns the current RPM of the motor to two decimal places
 float StepperMotor::getMotorRPM() {
-    return (getEncoderSpeed() / 360);
+
+    // Convert getEncoderSpeed() (in deg/s) to RPM
+    return (getEncoderSpeed() * 60 / 360);
```

Comment:
```
return (getEncoderSpeed() / 6);
```

Reply:
It will do it itself. Any type of math equations that can be done early will be. You can test it by using both and seeing that the number of instructions doesn't change. That's why I moved the SINE_POWER to the log2(). It doesn't have a performance impact, so why not? It'll prevent future issues if the value is ever changed